runtime/src/bank.rs

```diff
@@ -1706,11 +1707,8 @@ mod tests {
             &dummy_leader_pubkey,
             dummy_leader_lamports,
         );
-        genesis_config.rent = Rent {
-            lamports_per_byte_year: 5,
-            exemption_threshold: 1.2,
-            burn_percent: 5,
-        };
+
+        genesis_config.rent = Rent::new(5, 1.2, 5);
```

Comment:
I don't think this is a readability improvement.  We go from a list of descriptive name/value pairs to an unreadable string of 3 numbers.